tornado/web.py

```diff
@@ -1978,6 +1978,16 @@ def find_handler(self, request, **kwargs):
 
     def get_handler_delegate(self, request, target_class, target_kwargs=None,
                              path_args=None, path_kwargs=None):
+        """Returns `~.httputil.HTTPMessageDelegate` that can serve a request
+        for application and `RequestHandler` subclass.
+
+        :arg httputil.HTTPServerRequest request: current HTTP request.
+        :arg RequestHandler target_class: a `RequestHandler` class.
+        :arg dict target_kwargs: keyword arguments for ``target_class`` constructor.
+        :arg list path_args: positional arguments for ``target_class`` HTTP method that
+            will be executed while handling a request (``get``, ``post`` or any other).
+        :arg dict path_kwargs: keyword arguments for ``target_class`` HTTP method.
+        """
```

Comment:
I believe it's exposed automatically: https://github.com/tornadoweb/tornado/blame/master/docs/web.rst#L146 as a member of Application class